components/database/mongodb-4/mongodb-40/files/mongodb-4.xml

```diff
@@ -15,7 +15,7 @@ Copyright 2012 EveryCity Ltd. All rights reserved.
 -->
 <service_bundle type="manifest" name="mongodb">
 
-  <service name="application/database/mongodb_40" type="service" version="1">
+  <service name="application/database/mongodb-4" type="service" version="1">
```

Comment:
I'm not sure it's safe. It's fine to replace 4.0 with 4.4, but don't call it an update - it's intrusive action, which should require manual decision, not just 'pkg update' magic. There also can be reasons to run 4.0 and 4.2 (or 4.4) in parallel - for example, for 4.0->4.4 update. 
Also, why do we need such complicated hierarchy database/mongodb-4/mongodb-40?
Perhaps, just introduce database/mongodb-44 and mark 40 obsolete a bit later (and no, it's not rename)?

Reply:
That was an error with 'git mv'.
So we go back now to 4.4?

Reply:
Perhaps, I'm late for the party, but basically yes. Suppose, you update your production server and get 4.4 instead of 4.0. What will you feel?

Reply:
In this case we need a new folder mongo-44.
I could create a new folder mongo-44, move all files to the new folder and squash the commits.

Reply:
I think you can avoid touching 4.0 for some time. At least one should announce that it's going to disappear. I hope there are people running OI in production ;)